pkg/cmd/publish-provisional-artifacts/main.go

```diff
@@ -184,8 +184,8 @@ func run(
 			} else {
 				licenseFiles := []release.ArchiveFile{
 					{
-						LocalAbsolutePath: filepath.Join(o.PkgDir, "licenses", "LICENSE.txt"),
-						ArchiveFilePath:   "LICENSE.txt",
+						LocalAbsolutePath: filepath.Join(o.PkgDir, "licenses", "LICENSE"),
```

Comment:
```suggestion
						LocalAbsolutePath: filepath.Join(o.PkgDir, "LICENSE"),
```

Reply:
Ah! Yes. I hadn't gotten to debugging that yet. Thanks.